### PR TITLE
Feat/vue range input

### PR DESCRIPTION
**Pull request by anik-ghosh-au7**
**Before submitting a pull request,** please make sure the following is done:

- [x] Describe the proposed changes and how it'll improve the library experience.
    - RangeInput com ponent added to vue-searchbox library
    
- [x] Please make sure that there are no linting errors in the code.

- [x] Add a demo video/gif/screenshot to explain how did you test the fix.
https://www.loom.com/share/1db133f8c61047daafbd4dba05dd0047

- [] If it is a global change, try to add any side effects that it could have.

- [x] Create a PR to add/update the docs (if needed) at [here](https://github.com/appbaseio/Docs).
https://github.com/appbaseio/Docs/pull/195

- [x] Create a PR to add/update the storybook (if needed) at [here](https://github.com/appbaseio/playground).
https://github.com/appbaseio/vue-playground/pull/28

**Learn more about contributing:** [Contributing guides](https://github.com/appbaseio/reactivesearch/blob/next/.github/CONTRIBUTING.md)

### CODE REVIEW

**Review by bietkul — APPROVED** [review]

File: packages/vue/src/components/range/RangeInput.jsx [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+					filterLabel={filterLabel}
+					react={react}
+					showFilter={showFilter}
+					showCheckbox={showCheckbox}
+					title={title}
+					URLParams={URLParams}
+					sliderOptions={sliderOptions}
+					nestedField={nestedField}
+					on-change={this.handleOnChange}
+					on-value-change={this.handleValueChange}
```
Comment (bietkul) [inline packages/vue/src/components/range/RangeInput.jsx:216]: @anik-ghosh-au7 You can remove this. We have fixed the issue with `change` event in RangeSlider.

**Review by bietkul — CHANGES_REQUESTED** [review]

File: packages/vue/src/components/range/RangeInput.jsx (hunk 1 above)
Comment (anik-ghosh-au7) [reply packages/vue/src/components/range/RangeInput.jsx:216]: @bietkul these are needed, it is how the listeners are passed to the `RangeSlider` component

**Comment by anik-ghosh-au7** [comment]
@bietkul  I have made the changes and run the example after pulling the latest from next

File: packages/vue/src/components/range/RangeInput.jsx [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+				/>
+				<Flex className={getClassName(innerClass, 'input-container') || null}>
```
Comment (bietkul) [inline packages/vue/src/components/range/RangeInput.jsx]:
```suggestion
				<Flex class={getClassName(innerClass, 'input-container') || null}>
```
`className` doesn't work for vue.

File: packages/vue/src/components/range/RangeInput.jsx [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+							key={this.$data.currentValue.start}
+							name="start"
+							type="number"
+							onChange={this.handleInputChange}
+							value={this.$data.currentValue.start}
+							step={stepValue}
+							themePreset={themePreset}
+							aria-label={`${componentId}-start-input`}
+							min={this.$props.range ? this.$props.range.start : 0}
+							className={getClassName(innerClass, 'input') || null}
```
Comment (bietkul) [inline packages/vue/src/components/range/RangeInput.jsx]:
```suggestion
							class={getClassName(innerClass, 'input') || null}
```

File: packages/vue/src/components/range/RangeInput.jsx [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+							key={this.$data.currentValue.end}
+							name="end"
+							type="number"
+							onChange={this.handleInputChange}
+							value={this.$data.currentValue.end}
+							step={stepValue}
+							themePreset={themePreset}
+							aria-label={`${componentId}-end-input`}
+							max={this.$props.range ? this.$props.range.end : 10}
+							className={getClassName(innerClass, 'input') || null}
```
Comment (bietkul) [inline packages/vue/src/components/range/RangeInput.jsx]:
```suggestion
							class={getClassName(innerClass, 'input') || null}
```

**Review by bietkul — CHANGES_REQUESTED** [review]

**Pull request merged**